components/server/ee/src/user/user-service.ts

```diff
@@ -78,7 +85,8 @@ export class UserServiceEE extends UserService {
 
         // 1. check the license
         const userCount = await this.userDb.getUserCount(true);
-        if (!this.licenseEvaluator.hasEnoughSeats(userCount)) {
+        if (userCount > -1) {
+            //!this.licenseEvaluator.hasEnoughSeats(userCount)) {
             const msg = `Maximum number of users permitted by the license exceeded`;
```

Comment:
I'd word this slightly more specifically:
```suggestion
            const msg = `Unable to log in. The license used for this installation does not have enough seats. Please contact your administrator to increase the number of seats for your license.`;
```

Reply:
**suggestion**: Also, it'd be great if we could surface this CTA to request a new license for the admins when browsing the dashboard.